unobstructed_lanechange/config/parameters.yaml

```diff
@@ -65,4 +65,8 @@ enable_object_avoidance_lc : false
 
 # Double: The minimum timestep used for planning trajectory
 #Units: seconds
-min_timestep : 0.1
\ No newline at end of file
+min_timestep : 0.1
+
+# Double : The additional downtrack beyond requested end dist used to fit points along spline. Extra points removed before returning trajectory. Feature not used if end dist is route end
+#Units: meters
+ending_buffer_downtrack : 5.0
```

Comment:
Still have unobstructed_lane change package changes in this and the launch file.
The launch file change actually looks correct, but I don't know if its functional. Please check with @adev4a